lib/assesments/assessment_2/fi-as2-hira/Screens/Home/widgets/fi_as2_hira_bottom.dart

```diff
@@ -0,0 +1,107 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_interns/assesments/assessment_2/fi-as2-hira/Screens/My%20prpofile/fi_as2_hira_profile.dart';
+import 'package:flutter_interns/assesments/assessment_2/fi-as2-hira/constants/fi_as2_hira_colors.dart';
+import 'package:flutter_interns/assesments/assessment_2/fi-as2-hira/widgets/design/fi_as2_hira_sample.dart';
+
+class H1Bottombar extends StatefulWidget {
```

Comment:
BottomBar is a core widget of the application it shouldn't be in home widgets only